crnk-core/src/main/java/io/crnk/core/engine/internal/dispatcher/controller/ResourceDeleteController.java

```diff
@@ -51,4 +66,50 @@ public boolean isAcceptable(JsonPath jsonPath, String method) {
 			return response;
 		});
 	}
+
+	protected Serializable getParsedId(Resource dataBody, ResourceInformation resourceInformation) {
+		String id = dataBody.getId();
+		return resourceInformation.parseIdString(id);
+	}
+
+	protected List<Resource> getRequestBodys(Document requestDocument, JsonPath path, HttpMethod method) {
```

Comment:
relationship-focused controllers also make use of ResourceIdentifier collections, maybe there is some potential to share some of those helper methods. Most notable the PATH/POST/DELETE of relationships.